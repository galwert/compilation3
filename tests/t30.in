// this is oke :)
void printOK() {
    print("OK!");
    printi(5);
}

void main()
{
    int x = (((((5)))));
    byte Bro = 5 b; // really <Partner>?

    x = (((((x + Bro)+x)+x)));
    int Bro2 = (((((((((x+Bro))))))+1))+3);
    
    // we have some explicit casts here!! make sure you allow dis
    byte Bro3;
    Bro3 = (byte)x;
    x = (int)Bro3;
}