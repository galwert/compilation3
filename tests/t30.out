---end scope---
---end scope---
x INT 0
Bro BYTE 1
Bro2 INT 2
Bro3 BYTE 3
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
printOK ()->VOID 0
main ()->VOID 0
